DEVELOPMENT.md

```diff
@@ -30,28 +36,69 @@ This requires that `kind` is installed and that either `podman` or `docker` is a
 > ```
 
-> [!note]
-> In its current state the local cluster is created to be either SC or WC, and currently the scripts does not support setting up both a SC and WC.
-> To setup two independent clusters the following must be changed manually:
->
-> 1. the kind-config/local-cluster-profile must not bind the ingress controller to the same address, and
-> 1. the node-local-dns/local-resolve config must be updated to point towards the correct clusters.
-
 > [!tip]
-> Since local clusters are effectively ephemeral they can pull a lot of images and `kind` has no build in system to manage images.
+> Since local clusters are effectively ephemeral they can pull a lot of images and `kind` has no built-in system to manage images.
 > So, for the local clusters script there are commands to create and delete local pull through registry caches for a few upstream registries.
-> Commands to do so is `./scripts/local-cluster.sh cache <create|delete>`, then one can make use of the local cluster profiles `<single|multi>-node-cache` that are prepared to use it by default.
+> The command to do so is `./scripts/local-cluster.sh cache <create|delete>`, then one can make use of the local cluster profiles `<single|multi>-node-cache` that are prepared to use it by default.
+
+As a first step, configure the clusters:
 
 ```sh
 # with CK8S_CONFIG_PATH and CK8S_PGP_FP set
 ./scripts/local-cluster.sh config <name> <apps-flavor> <domain>
-./scripts/local-cluster.sh create <name> <kind-config|local-cluster-profile> # use list profiles to see built-in configs
 ```
 
-This will configure apps with `ck8sCloudProvider: none` and `ck8sFlavor: <apps-flavor>` and set some default values to run on local clusters.
-By default it will use `calico` for networking, `local-path-provisioner` for block storage, and `minio` for object storage.
+This will use the default local listen address of `127.0.64.43` for the SC worker node and `127.0.64.143` for the WC worker node.
+
+Apps will be configured with `ck8sCloudProvider: none` and `ck8sFlavor: <apps-flavor>` and set some default values to run on local clusters.
+By default, local clusters will use `calico` for networking, `local-path-provisioner` for block storage, and `minio` for object storage.
+
+We'll also be passing the `--skip-minio` options for the WC, to deploy block storage only to the SC cluster.
+
+```sh
+./scripts/local-cluster.sh create <name>-sc <kind-config-file|local-cluster-profile>
+./scripts/local-cluster.sh create <name>-wc <kind-config-file|local-cluster-profile> --skip-minio
+```
+
+> [!important]
+> To support the integrated SC <--> WC scenario we require that the cluster names be suffixed with either `-sc` or `-wc`
+
+#### In-cluster DNS setup
+
+Making the clusters "aware" of each other will require custom DNS configuration snippets to both the SC and WC that ensure:
+
+- `(grafana|harbor|opensearch|ops).<domain>` resolve to an SC worker node IP from within the WC cluster
```

Comment:
These subdomains are configurable, not sure if that should be handled by this or if it's enough leave a note that it's not supported?